packages/utils/src/modify-config-helper.ts

```diff
@@ -84,7 +85,7 @@ export function modifyHelperUtil(
             }
         }
     } catch (err) {
-        console.error(red('\nYour package.json was incorrectly formatted.\n'));
+        logger.error('\nYour package.json was incorrectly formatted.');
```

Comment:
We are using extra `\n` and using `console.error`, so extra newline here expected here